hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/lock/TestKeyPathLock.java

```diff
@@ -218,26 +215,15 @@ private void testDiffKeyPathWriteLockMultiThreadingUtil(
       OzoneManagerLock lock, CountDownLatch countDown,
       String[] sampleResourceName) {
 
-    lock.acquireWriteHashedLock(resource,
-        generateResourceHashCode(resource, sampleResourceName));
+    lock.acquireWriteLock(resource, sampleResourceName);
     LOG.info("Write Lock Acquired by " + Thread.currentThread().getName());
 
     // Waiting for all the threads to be instantiated/to reach
     // acquireWriteHashedLock.
     countDown.countDown();
-    while (countDown.getCount() > 0) {
```

Comment:
Nit: can we add `countDown.await()` here

Reply:
For this aged PR, I think we should keep it as light as it is now. Any improvements in the existing test can be done separately.

Reply:
Sure, thanks @duongkame!